src/tests/specs/common.ts

```diff
@@ -50,3 +87,83 @@ function getRootDir() {
     return path.join(__dirname, "..", "..", "..");
   }
 }
+
+export class TestContext {
```

Comment:
`TestContext` is an object that stores a bunch of values across all the tests of a given spec. It's useful to reduce the amount of boilerplate. Not super happy about it for now.